webgui/scripts/codecompass/view/component/Text.js

```diff
@@ -198,7 +198,8 @@ function (declare, domClass, dom, style, query, topic, ContentPane, Dialog,
     //--- Get AST node info ---//
 
     var service = model.getLanguageService(fileInfo.type);
-    var astNodeInfo = service.getAstNodeInfoByPosition(fpos);
+    if (service.getAstNodeInfoByPosition)
```

Comment:
How can this happen? By the way `getLanguageService()` will always return a language service which has a `getAstNodeInfoByPosition()` function, or a falsy value. This falsyness should be rather checked if necessary (`if (service) ...`).